app/controllers/comments_controller.rb

```diff
@@ -34,7 +45,8 @@ def comment_params
       .permit(:content, :task_id)
       .merge(
         {
-          user: current_user
+          user: current_user,
+          task_id: params[:task_id]
```

Comment:
why? https://github.com/DmitryBarskov/task-tracker/pull/64/files#diff-34e52388abd1ae0658edd56e12cd6d935fac98673361084ba07a99ca4bb836e7R45

Reply:
ID is already in params, isn't it?

Reply:
Without this, the comment has no task_id